forge-game/src/main/java/forge/game/Game.java

```diff
@@ -860,11 +866,23 @@ public void onPlayerLost(Player p) {
         // 901.6: If the current planar controller would leave the game, instead the next player
         // in turn order that wouldn't leave the game becomes the planar controller, then the old
         // planar controller leaves
+        if (planarControllerLost) {
+            for (Card c : getActivePlanes()) {
+                if ((c != null) && !(c.getOwner().equals(p))) {
+                    c.setController(getNextPlayerAfter(p), 0);
+                    getAction().controllerChangeZoneCorrection(c);
+                }
+            }
+        }
         // 901.10: When a player leaves the game, all objects owned by that player except abilities
         // from phenomena leave the game. (See rule 800.4a.) If that includes a face-up plane card
         // or phenomenon card, the planar controller turns the top card of his or her planar deck face up.the game.
-        if (planarControllerLost) {
-            getNextPlayerAfter(p).initPlane();
+        if (planarOwnerLost) {
+            Player planarController = getPhaseHandler().getPlayerTurn();
+            if (planarController.equals(p)) {
+                planarController = getNextPlayerAfter(p);
+            }
+            planarController.planeswalkTo(null, new CardCollection(planarController.getZone(ZoneType.PlanarDeck).get(0)));
```

Comment:
from looking at the rules I'd say you should still call `planarController.leaveCurrentPlane()` before

Reply:
I've added logic to create a PlaneswalkedFrom trigger, though avoiding actually using leaveCurrentPlane for the purposes of an edge case with Norn's Seedcore (where there might be face-up planes owned by multiple players; in that case, from my understanding, only those actually owned by the player leaving the game should be planeswalked away from, while leaveCurrentPlane will put _all_ active planes on the bottom of their owners' planar decks)